maven-compat/src/main/java/org/apache/maven/repository/MetadataResolutionRequest.java

```diff
@@ -108,14 +108,14 @@ public MetadataResolutionRequest setLocalRepository( ArtifactRepository localRep
         return this;
     }
 
-    public List<ArtifactRepository> getRemoteRepostories()
+    public List<ArtifactRepository> getRemoteRepositories()
```

Comment:
This is probably an api change.
I can provide deprecation markers and/or split this into a distinct PR (applies for all similar items)